### DISCUSSION TITLE
list of the best VPN for people in Iran.

### DISCUSSION THREAD

**Discussion by foad-heidari** [WeAreMahsaAmini/Core#160] [Ideas]
most of the people don't know about which vpn is the best and also working always. 
maybe we can have a page to show all the best VPN out there,

also maybe we can create our own vpn?

**Comment by AnonymousMaximus2** [comment]
I can help on creating private VPN network that people can downloads and setup on their mobile app, 

1- Server installation,
2- Open VPN server installation on world dataenters
3- VPN application development 
4- uploading application on Google play store

also I suggest to take a look at this discussion:

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Reply by baharehoseini** [reply]
Unfortunately, OpenVPN doesn't work well since yesterday. I bought ExpressVpn for them it works well (13$ per month) and 5  people can use it.

**Comment by AnonymousMaximus2** [comment]
OpenVPN does not means accessible from everywhere,

Open VPN is a opensource projet that can be edited customized and manipulated source for custom usage,

by using OPENVPN source we can develop our own VPN infrastructure

**Reply by foad-heidari** [reply]
we can have the openVPN as our VPN and also show the best VPNs out there.

i don't know. a server how much people can handle, we can have multiple servers and setup VPN proxy on them

**Comment by AnonymousMaximus2** [comment]
A normal VPS server with 1 GB of ram and 1 GB of HDD and 1 core CPU can handle more than 1 thousands of VPN accounts, 

the main thing is Network Port that must be high as possible, 

in most advanced datacenters you can rent VPS with more than 1 Gbit of network connection.

the VPN server does not needs much ram or CPU or storage

**Reply by foad-heidari** [reply]
okay so what will block us to do it?

**Comment by hiddify-com** [comment]
OpenVPN is not good because it cannot separate Iranian websites and others
Clash is the better option. You can specify the GeoIP.
And it is undetectable since it uses the TLS protocol
see my repository
https://hiddify.github.io/filter.html

By integrating GoodByeDPI, DNS over HTTPS and clash we can have maximum ability to prevent to be detected and to maximizing the speed

**Comment by AnonymousMaximus2** [comment]
Psiphon 3 is working fine in Iran, 

Tested on Android and windows platforms.

**Comment by IranZop** [comment]
#249

**Comment by aslanradin-debug** [comment]
Guys I'm an Iranian GitHub is now open in iran in this blackout can someone give me vpn? I have my family and many people out there need vpn to connect open world freedom but ye give me vpn I can upload to Iranian sites and everyone can download
